DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/ManageCourseController.cs

```diff
@@ -101,6 +102,68 @@ EditLearningPathwayDefaultsViewModel model
             return RedirectToAction("Index", new { customisationId = model.CustomisationId });
         }
 
+        [HttpGet]
+        [Route("CourseDetails")]
+        public IActionResult EditCourseDetails(int customisationId)
+        {
+            var centreId = User.GetCentreId();
+            var categoryId = User.GetAdminCategoryId()!;
+
+            var courseDetails = courseService.GetCourseDetailsForAdminCategoryId(
+                customisationId,
+                centreId,
+                categoryId.Value
+            );
+
+            var model = new EditCourseDetailsViewModel(courseDetails!);
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [Route("CourseDetails")]
+        public IActionResult SaveCourseDetails(
+            int customisationId,
+            EditCourseDetailsViewModel model
+        )
+        {
+            if (customisationId != model.CustomisationId)
+            {
+                return StatusCode(500);
+            }

```

Comment:
Yeah, here is an example why we should perhaps split to ViewModel/FormData, we don't want to post a value for the customisationId.